### ISSUE TITLE
Does not work with PS Nova Cart on 3.68

### ISSUE THREAD

**Issue by the-importer** [dots-tb/rePatch-reDux0#21]
You said 15 days ago in #14 that you would reopen the case if your next version didn't fix the issue, will the Alpha version you released 6 days doesn't work, still can't use patches with physical cards.

**Comment by dots-tb** [comment]
The issue of #14 was that it softlocked on 3.68. Your issue is different. Please check your tai config.txt.

**Comment by the-importer** [comment]
Well in my issue (#9), you linked #7 and #14 was linked in that last one, so I assumed that everything was part of the same issue. Anyway, I showed you screenshots of my config file, you know everything was well configured.

**Comment by dots-tb** [comment]
Oh yeah, alright. Well, did you do that eboot test?

**Comment by the-importer** [comment]
@dots-tb 
Ya, I did do the test, but you closed the topic before I could post the picture. Here, I did the test again again: https://s8.postimg.cc/8s2fwsqid/2018-07-30-211050.jpg

**Comment by the-importer** [comment]
The config and plugins are in ur0:tai
The game updates and DLC are in ux0 in their appropriate folders
The games themselves are physical

**Comment by dots-tb** [comment]
I'm asking if the file that you are trying to patch actually exists within app (the one on the cart) or patch (maybe on the cart if its writable or on ux0).

**Comment by the-importer** [comment]
I would assume yes because if I install PSNOVA as VPK, update the game with other VPK files and then replace the NOVA_FileList_Vita_patch.cpk inside the app folder for the translated one, then it works.

**Comment by dots-tb** [comment]
Please, when you can, check if the file is in the patch folder.

**Comment by the-importer** [comment]
Official 1.05 update
https://s8.postimg.cc/4e35c67hh/2018-07-31-005312.jpg

Translation patch
https://s8.postimg.cc/pnqrn0vhx/2018-07-31-005410.jpg

**Comment by dots-tb** [comment]
Can you try renaming the original file in the official patch folder and seeing if that crashes the game?

**Comment by the-importer** [comment]
https://s8.postimg.cc/f7ankt6t1/2018-07-31-014045.jpg

**Comment by the-importer** [comment]
@dots-tb 
Hi, just to let you know, I hooked a 2nd PSTV that I have that's on firmware 3.60, did the same setup and it works, so it's most likely a firmware issue.

**Comment by dots-tb** [comment]
https://github.com/dots-tb/rePatch-reDux0/wiki/Debug-Info-for-Bugs-and-Modding @the-importer I've added a page for getting log info. Please check it out.

**Comment by the-importer** [comment]
@dots-tb 
Me again, I was googling to see if there was a rePatch version for the translation of I Am Setsuna since I have a physical card for the game as well and found one. It was originally made for the old MaiDump version of the game, but someone made a rePatch version: 
https://www.reddit.com/r/VitaPiracy/comments/8ct8px/i_am_setsuna_english_repatch_for_nonpdrm_repacked/
This actually works with my firmware 3.68, only different between this patch and PSNOVA/PSO2 is that it's not simply a CPK file, there's more to it.

**Comment by the-importer** [comment]
@dots-tb 
Sorry, but I'll need more info in order to setup ShipLog. It says "Install ShipLog.skprx in taiHEN config.txt"
Where's this ShipLog.skprx file?
Does taiHen = tai or is it somerthing different?

**Comment by dots-tb** [comment]
Yeah, I finally added the bug tag. Just let me know if you are able to getting logging information out of it.

**Comment by the-importer** [comment]
@dots-tb 
It says to type this command on my PC "nc -l -p 3333"
Tried in a CMD prompt, doesn't recognized the program.

**Comment by dots-tb** [comment]
If you don't want to use net, you can just create ux0:dump and then open the app and DUMP LOG BUFFER TO DISK and it will create logger.txt in that folder. "nc" is netcat, there are some windows version compiled on google or if you are on linux you can install it through your package manager software.

**Comment by the-importer** [comment]
@dots-tb 
Is this any good?
[logger.txt](https://github.com/dots-tb/rePatch-reDux0/files/2247921/logger.txt)

**Comment by dots-tb** [comment]
Can you launch a working game and then the non-working game? Or did you do that?

**Comment by the-importer** [comment]
@dots-tb 
I followed the instructions which didn't say anything about running games. You want me to run PSNOVA and then I Am Setsuna?

**Comment by dots-tb** [comment]
Well the instructions are just for setting up the logger. I would like that, yes.

**Comment by the-importer** [comment]
OK, so I delete the logger file, booted PSNOVA and then I Am Setsuna. Here's the new file, but it doesn't look like it recorded any of the games I booted.
[logger.txt](https://github.com/dots-tb/rePatch-reDux0/files/2248128/logger.txt)

**Comment by dots-tb** [comment]
Are you using the debug version of rePatch on the wiki page? It should log every file call seen by rePatch.

**Comment by the-importer** [comment]
No to both questions. In case it wasn't clear, I don't know anything about what you're making me do. I have no issues following instructions when it comes to computer stuff, but if those said instructions aren't clear or are lacking certain steps, then it's wasting both of our time. Is there anything else I need to know before doing this test again? Anything that wasn't in those instructions you've sent me?

**Comment by TheRadziu** [comment]
@the-importer you need to replace regular repatch with debug repatch linked on wiki dots made.

that means this build: https://transfer.sh/7cJYQ/repatch.skprx

**Comment by the-importer** [comment]
@dots-tb @TheRadziu 
OK, this log file makes more sense: [logger.txt](https://github.com/dots-tb/rePatch-reDux0/files/2249547/logger.txt)

**Comment by Aida-Enna** [comment]
Both the PSO2 Vita English patch and the PS Nova English patch work on 3.68. They've both been tested by multiple people and confirmed working.

**Comment by the-importer** [comment]
@RikuNoctis
Are you using a physical card or a rip?

**Comment by RikuNoctis** [comment]
@the-importer In my case, both of them.
The NoNpDrm version on my OLED Vita. The cart version on my slim Vita (I don't have SD2Vita on this one).

**Comment by the-importer** [comment]
@Riku-Noctis
Well it fails to load on both my Vita-1000 and PSTV on 3.68, but loads properly on my PSTV that's on 3.60. My repatch setup is correct since I can load the translation of I am Setsuna.

**Comment by Aida-Enna** [comment]
I'm still on 3.65 personally, but people with both the cart and rip have confirmed it for me.

**Comment by the-importer** [comment]
@Aida-Enna
I guess I can update my PSTV that's on 3.60 to 3.65 and see if that still works, that would help pin-point the issue.

**Comment by the-importer** [comment]
@dots-tb 
After reading that some people made this game work with h-encore and 3.67/3.68, I decided to see if my computer troubleshooting skills could solve this puzzle and indeed to were able to. I knew that there was   nothing wrong with the patches or the way I setup everything and if the firmware wasn't the issue, then what else was different with my PSTV and Vita on firmware 3.68 and my old one with firmware 3.60? Well, the memory card. 

You see, I tried the patch on my 2nd PSTV using the internal 1GB of internal memory which was freshly formatted. On the other hand, on my main PSTV and my Vita 1000, I just swap my 32GB memory card between the 2 systems. That being said, I have a 16GB memory card which I stopped using, but still had some stuff on it. I inserted into my 2nd PSTV (3.60), copied the needed file from the internal storage and tried PS NOVA, the patch didn't work. So I formatted this card, had the system automatically copied to files from the internal storage to the 16GB memory card and tried PS NOVA again, it worked!!!

Now I have yet to format my 32GB card since I'll need to back a few things up first, but I'm 90% certain that this will work. That being said, I have no clue with I need to do this in the first place. It's not like rePatch wasn't working at all, the patch for I am Setsuna was working. My theories:

1- A pure coincidence that 2 memory cards had that problem (I don't personally believe in coincidences when it comes to computers)
2- A save data, patch or app on the memory cards was interfering with the patch for PS NOVA and PSO2
3- Perhaps the fact that the memory cards were linked to 2 accounts using Simple Account Switcher (the most likely culprit) was causing the incompatibility

I'll do some tests on the 16GB memory card and report back.

**Comment by Aida-Enna** [comment]
Nice to hear you got it working! And when in doubt, blame SilicaAndPina (Author of Simple Account Switcher).

**Comment by the-importer** [comment]
@dots-tb @Aida-Enna
Well, everything is working fine now. That being said, I still do not know what caused the issue in the first place on both my memory cards. I've restored all of my save data, patches and DLC, h-encore is running fine, I've installed the 4 homebrews that I need are installed: VitaShell, Adrenaline, Simple Account Switcher & VitaRW and everything in my ur0 folder in both my Vita and PSTV remain the same (config.txt, all of the suprx and skprx files).

Only thing I can think of would be homebrews that I had installed, stopped using and removed, but some traced remained (Vitamin, MaiDump, etc..). So if this happening to you, I would suggests backing up your save data using QCMA, formatting your memory card, restore your data install the essentials (h-encore, VitaShell, Adrenaline) and see if the problems persists. If everything is working, then install other homebrews gradually and test. If one of your homebrews brakes rePatch for any game, then there's your culprit.

Sorry dots-tb if this has cause any inconveniences, never figured I'd see the days where I would need to apply a PC solution for a game system. I guess it's true what they say, these game systems are nothing but underpowered personal computers these days.
